src/assets/scss/styles.scss

```diff
@@ -0,0 +1,445 @@
+
+.about {
+  padding: 125px 0 165px 0;
+  text-align: center;
+  background-color: #f8fafa;
+  &__title {
+    font-size: 38px;
+    color: $t-primary;
+    font-weight: bold;
+    padding-bottom: 5px;
+    text-transform: uppercase;
+    letter-spacing: 1.5px;
+  }
+  &__text {
+    color: #5e5f5f;
+    font-family: "Merriweather";
+    font-size: 18px;
+    text-transform: uppercase;
+    line-height: 1.5;
+    letter-spacing: 1.3px;
+  }
+  &__content {
+    padding-top: 116px;
+    display: flex;
+    justify-content: space-around;
+  }
+  &__img {
+    margin-left: 33px;
+  }
+}
+
+.description {
+  width: 490px;
+  text-align: left;
+  padding-top: 29px;
+  margin-left: 85px;
+
+  &__title {
+    font-family: "Merriweather";
+    font-size: 20px;
+    font-weight: bold;
+    text-transform: uppercase;
+    padding-bottom: 23px;
+    letter-spacing: 1.4px;
+  }
+  &__text {
+    font-family: "Raleway";
+    font-size: 16px;
+    line-height: 1.75;
+    padding-bottom: 27px;
+    letter-spacing: 0.8px;
+    color: #929393;
+  }
+  &__text:nth-of-type(2) {
+    padding-bottom: 34px;
+  }
+
+  &__subtitle {
+    font-family: "Merriweather";
+    font-size: 17px;
+    font-weight: bold;
+    text-transform: uppercase;
+    padding-bottom: 18px;
+    letter-spacing: 1.2px;
+  }
+}
+@media (max-width: 1169px) {
+  .about__img {
+    margin-left: 0px;
+  }
+  .description {
+    margin: 0;
+  }
+}
+@media (max-width: 1023px) {
+  .about__content {
+    justify-content: space-between;
+    padding-top: 50px;
+  }
+  .about__img {
+    width: 400px;
+  }
+  .description {
+    width: 400px;
+
+    &__title {
+      line-height: 1.5;
+    }
+  }
+}
+@media (max-width: 767px) {
+  .about {
+    padding: 70px 0 0;
+  }
+  .about__title {
+    text-align: center;
+    line-height: 1.5;
+  }
+  .about__content {
+    padding-top: 30px;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  }
+}
+@media (max-width: 500px) {
+}
+@media (max-width: 425px) {
+  .about__title {
+    font-size: 30px;
+  }
+  .about__img {
+    max-width: 100%;
+  }
+
+  .description {
+    width: 100%;
+
+    &__title {
+      text-align: center;
+    }
+    &__subtitle {
+      text-align: center;
+    }
+    &__text {
+      text-align: justify;
+    }
+    &__content {
+      padding-top: 50px;
+    }
+  }
+}
+/* ------- Next section work  ------------*/
+
+.work {
+  padding: 100px 0 110px 0;
+  text-align: center;
+
+  &__title {
+    padding-bottom: 10px;
+    font-size: 38px;
+    color: $t-primary;
+    font-weight: bold;
+    text-transform: uppercase;
+    letter-spacing: 1.5px;
+    letter-spacing: 1.9px;
+  }
+  &__text {
+    color: #5e5f5f;
+    font-family: "Merriweather", serif;
+    font-size: 17px;
+    text-transform: uppercase;
+    letter-spacing: 0.5px;
+    letter-spacing: 1.2px;
+  }
+  &__content {
+    padding-top: 113px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0 -8px;
```

Comment:
You have made something bad if you need negative margins. Try to correct it here and next